### PR TITLE
Feature/nuc 2328 disable platform specific template generation

### PR DESCRIPTION
**Pull request by osmanertem**

### CODE REVIEW

File: generator/templates/nvw/src/App.vue [hunk 1]
```diff
@@ -135,7 +115,7 @@
 <%_ if (rootOptions.cssPreprocessor) { _%>
 <%_   if (rootOptions.cssPreprocessor == 'sass' || rootOptions.cssPreprocessor == 'scss'  || rootOptions.cssPreprocessor == 'dart-sass' ) { _%>
 <%#   -------------------- IS Using sass, scss OR dart-sass -------------------- -%>
-<style web lang="scss">
+<style lang="scss">
```
Comment (mylmz10) [inline generator/templates/nvw/src/App.vue:118]: Could you change nvw prefix to vxp?

File: generator/templates/nvw/src/App.vue [hunk 2]
```diff
@@ -21,17 +11,7 @@
...
-    <GridLayout rows="auto, auto">
-      <!-- copy-webpack-plugin copies asset from src/assets to project output/build directory /assets -->
-      <Img src="~/assets/logo.png" row="0" class="m-20"/>
-      <HelloWorld :msg="msg" row="1"/>
-    </GridLayout>
-  </Page>
-</template>
-<template native>
+<template>
   <Page>
```
Comment (mylmz10) [inline generator/templates/nvw/src/App.vue:15]: Could you change Image to VxpImage

Comment (osmanertem) [reply generator/templates/nvw/src/App.vue:15]: fixed

File: generator/templates/nvw/src/App.vue (hunk 1 above)
Comment (osmanertem) [reply generator/templates/nvw/src/App.vue:118]: fixed

**Pull request merged**